### ISSUE TITLE
Bug with crafting in bedrock

### ISSUE THREAD

**Issue by JustZakary** [GeyserMC/Geyser#3371]
### Describe the bug

On consoles, players aren't able to use the quick craft feature.

### To Reproduce

Go on Xbox
Try to craft using the recipe book.

### Expected behaviour

Expected for the item to be crafted and put in inventory.

### Screenshots / Videos

Video: https://www.reddit.com/r/GeyserMC/comments/yeg1eu/bug_when_console_players_try_to_craft/

### Server Version and Plugins

Geyser version 2.1.0-SNAPSHOT (git-master-ece1b2a)
Java: 1.19.1 - 1.19.2
Bedrock 1.19.0/1.19.2 - 1.19.40

### Geyser Dump

https://dump.geysermc.org/EbvhArbIvydDjyOZk2p7mfkgoDwVWLds

### Geyser Version

Geyser version 2.1.0-SNAPSHOT (git-master-ece1b2a)

### Minecraft: Bedrock Edition Device/Version

Minecraft Bedrock on Xbox One

**Comment by Latch93** [comment]
I'm getting this as well. Seems bedrock players crash when viewing the crafting menu, then clicking on the Recipe Book or other tabs. 
Message from Bedrock Player:
"I found out why it crashed when in bedrock.  If a bedrock player clicks any of the icons on the left, like the search thing or the grass block. you crash. Even if u click the chest to go back to your inventory."
TLDR: If a bedrock user clicks any of the "above" icons in the crafting menu, when they enter again to go in their inventory and click the chest, it crashes.

**Comment by Camotoy** [comment]
JustZakary, you need to update your Geyser version as we resolved this issue.
Latch, make sure you're on the latest Geyser version and if the issue persists then please open a new issue with replication steps. Crashing the game entirely may be a completely separate issue.
